tests/FSharp.Test.Utilities/ProjectGeneration.fs

```diff
@@ -1023,9 +1023,7 @@ type ProjectWorkflowBuilder
 
     member this.DeleteProjectDir() =
         if Directory.Exists initialProject.ProjectDir then
-            try
-                Directory.Delete(initialProject.ProjectDir, true)
-            with _ -> ()
+            () //try Directory.Delete(initialProject.ProjectDir, true) with _ -> ()
```

Comment:
Why are we not deleting?
Do we need a methodcall which is a conditional no-op?

Reply:
I thought I had uncommented that.  Perhaps it's on my laptop and I didn't push it!

Reply:
fixed